crates/l2/operator/l1_tx_sender.rs

```diff
@@ -1,3 +1,95 @@
+use std::str::FromStr;
+
+use ethereum_rust_blockchain::constants::TX_GAS_COST;
+use ethereum_rust_core::types::{EIP1559Transaction, TxKind};
+use ethereum_types::{Address, H256};
+use libsecp256k1::SecretKey;
+use tracing::{debug, warn};
+
+use crate::rpc::l1_rpc::L1Rpc;
+
+const RICH_WALLET_PK: &str = "0x385c546456b6a603a1cfcaa9ec9494ba4832da08dd6bcf4de9a71e4a01b74924";
+const RICH_WALLET_ADDR: &str = "0x3D1e15a1a55578f7c920884a9943b3B35D0D885b";
```

Comment:
Let's make a ticket to unhardcode this and more generally introduce some config files where we store all our configuration variables